.prettierrc

```diff
@@ -3,7 +3,7 @@
   "tabWidth": 2,
   "useTabs": false,
   "semi": true,
-  "singleQuote": true,
+  "singleQuote": false,
```

Comment:
We're currently using single quotes in our project. Was this change made for something I may not be aware of? You can see that multiple changes in this PR is your code changing from single quotes to double quotes.